src/main/java/seedu/cafectrl/data/Sales.java

```diff
@@ -56,4 +55,47 @@ public ArrayList<OrderList> getOrderLists() {
     public OrderList getOrderList(int index) {
         return orderLists.get(index);
     }
+
+    //@@author NaychiMin
+    /**
+     * Prints all sales data, organized by day, including dish names, quantities, and total cost prices.
+     *
+     * @param ui   The Ui object for user interface interactions.
+     * @param menu The Menu object representing the cafe's menu.
+     */
+    public void printSales(Ui ui, Menu menu) {
+        if (orderLists.isEmpty()) {
+            ui.showToUser("No orders for the day!");
+            return;
+        }
+
+        for (int day = 0; day < orderLists.size(); day++) {
+            OrderList orderList = orderLists.get(day);
+
+            ui.showToUser("Day " + (day+1) + ":");
+            ui.showToUser(String.format(HEADER_FORMAT, "Dish Name", "Dish Qty", "Total Cost Price"));
+            orderList.printOrderList(menu, ui);
+        }
+    }
+
+    /**
+     * Prints sales data for a specific day, including dish names, quantities, and total cost prices.
+     *
+     * @param ui  The Ui object for user interface interactions.
+     * @param menu The Menu object representing the cafe's menu.
+     * @param day The day for which sales data is to be printed.
+     */
+    public void printSaleByDay(Ui ui, Menu menu, int day){
```

Comment:
Perhaps add a whitespace here 
```suggestion
    public void printSaleByDay(Ui ui, Menu menu, int day) {
```